backends/ebpf/psa/ebpfPsaGen.cpp

```diff
@@ -632,11 +633,34 @@ bool ConvertToEBPFDeparserPSA::preorder(const IR::ControlBlock *ctrl) {
 
     if (ctrl->container->is<IR::P4Control>()) {
         auto p4Control = ctrl->container->to<IR::P4Control>();
-        // TODO: placeholder for handling digests
+        findDigests(p4Control);
         this->visit(p4Control->body);
     }
 
     return false;
 }
 
+void ConvertToEBPFDeparserPSA::findDigests(const IR::P4Control *p4Control) {
+    // Digests are only at ingress
```

Comment:
That's why you should probably give an error if you find one in egress.
I would check this later.
Also, perhaps this field should not be called "type"; we already have many types.